reddit/appclient.go

```diff
@@ -47,8 +90,14 @@ func (a *appClient) authorize() error {
 		a.cfg.app.Password,
 	)
 
-	a.baseClient.cli = cfg.Client(ctx, token)
-	return err
+	if err != nil {
+		return err
+	}
+
+	log.Printf("access_token generated: %s\n", token.AccessToken)
```

Comment:
Please remove debug line

Reply:
Not a debug line, real message logging.

Reply:
This package shouldn't log except with a user-provided logger. There should not be output that the user can't turn off.